src/node/blockstorage.cpp

```diff
@@ -94,12 +95,14 @@ void CleanupBlockRevFiles()
         remove(item.second);
     }
 }
+} // namespace node
 
 std::string CBlockFileInfo::ToString() const
```

Comment:
Would it be easier / cleaner to move this so it doesn't force a weird break in the namespace?